s2i-liberty/applier/templates/build.yml

```diff
@@ -0,0 +1,187 @@
+apiVersion: v1
+kind: Template
+labels:
+  template: liberty-builder
+metadata:
+  name: liberty
+objects:
+- apiVersion: v1
+  kind: ImageStream
+  metadata:
+    labels:
+      build: s2i-liberty
+    name: s2i-liberty
+  spec:
+    lookupPolicy:
+      local: false
+- apiVersion: v1
+  kind: ImageStream
+  metadata:
+    labels:
+      build: s2i-liberty
+    name: websphere-liberty
+  spec:
+    lookupPolicy:
+      local: false
+    tags:
+    - annotations:
+        openshift.io/imported-from: websphere-liberty:webProfile7
+      from:
+        kind: DockerImage
+        name: websphere-liberty:webProfile7
+      importPolicy: {}
+      name: webProfile7
+- apiVersion: v1
```

Comment:
why are we doing this vs just referencing imagestream/jboss-eap70-openshift in the openshift project?

Reply:
I'd say the real reason is probably because I was being lazy and only converted what was already in the existing template. I'll take a closer look or if @raffaelespazzoli might have some input since he did the original template.